doc/topics/cloud/vmware.rst

```diff
@@ -275,6 +276,10 @@ Set up an initial profile at ``/etc/salt/cloud.profiles`` or
             Enter the gateway for the network as a list. If the network specified
             is DHCP enabled, you do not have to specify this.
 
+        mac_address
+            Enter the mac address as a string. Note that is must be in the
+            00:50:56:00:00:00 - 00:50:56:3F:FF:FF range.
+
```

Comment:
It would also be good to specify here that this is optional. Please look at other directives for an example on how to specify it.